client/blocks/comment-detail/style.scss

```diff
@@ -383,11 +383,67 @@ a.comment-detail__author-more-element {
 }
 
 .comment-detail__reply {
-	line-height: 0;
+	// The inner working of these styles is covered here:
+	// http://alistapart.com/article/expanding-text-areas-made-elegant
+	.expanding-area {
+		min-height: 33px;
+		position: relative;
+		transition: min-height 0.2s ease-in-out;
+
+		&.focused {
+			min-height: 70px;
+		}
 
-	textarea {
-		border: none;
-		resize: vertical;
+		pre,
+		textarea {
+			font-family: $serif;
+			font-size: 14px;
+			line-height: 21px;
+			margin: 0;
+			max-height: 400px;
+			min-height: 33px;
+			overflow: hidden;
+			padding: 5px 60px 5px 5px;
+			white-space: pre-wrap;
+			word-wrap: break-word;
+		}
+
+		pre {
+			box-sizing: border-box;
+			display: block;
+			visibility: hidden;
+		}
+
+		textarea {
+			border: none;
+			height: 100%;
+			position: absolute;
+				left: 0;
+				top: 0;
+			resize: none;
+		}
+	}
+
+	.comment-detail__reply-submit {
+		color: lighten( $gray, 10% );
+		font-size: 12px;
+		font-weight: 600;
+		opacity: 0;
```

Comment:
I would have to admit that I can't tell the difference here. 😄  I suppose this is pretty minor either way, but I'd like us to get into the habit of not rendering things when they're not visible. With a lot of components, this can add up to a decent perf hit over time.